rsl/eval.py

```diff
@@ -408,6 +410,13 @@ def where(expr, selected):
             return value
             
         return lambda selected: where(node.expr, selected)
+
+    def accept_OrderedByNode(self, node):
+        if len(node.attributes):
+            if node.reverse:
+                return xtuml.reverse_order_by(*node.attributes)
```

Comment:
python sorted() function have reverse keyword, instead of two separate functions

Reply:
I know of the reverse parameter, however because we are passing around "order_by" as a single object (a compare function), it wasn't convenient to pass around a boolean value for reverse as well. I thought about doing it that way at first, but then we would need to pass two extra parameters to select_many. I suppose another way would be to create a new class which contained the compare function and the reverse flag, but I kind of like it the way I have it.